src/workers/helper.py

```diff
@@ -66,34 +67,33 @@ async def automod_thread(
     if starter_message is not None and time_since < timedelta(days=duration):
         return num_removed, num_errors
 
-    # delete thread and try to send DM to user
+    # delete thread and try to notify to user
     try:
         await thread.delete()
         logger.info(f"Thread {thread.id} has been deleted successfully")
         ret = num_removed + 1, num_errors
-        # TODO: uncomment DM portion and sleep when backlog is dealt with
-        # if thread.owner is not None:
-        #     try:
-        #         await send_dm(
-        #             thread.owner,
-        #             f'Your thread "{thread.name}" in <#{channel_id}> has been automatically deleted as {duration} days have passed without any activity or the starter message has been deleted.',
-        #         )
-        #     except discord.Forbidden:
-        #         raise UnableToDM("User has DMs disabled")
-        # else:
-        #     raise UnableToDM("User is not in the server")
+        if thread.owner is not None:
+            try:
+                await notifying_channel.send(
+                    f'{thread.owner.mention}, your thread "{thread.name}" in <#{channel_id}> has been automatically deleted as {duration} days have passed without any activity or the starter message has been deleted.'
```

Comment:
@clairefleuret text check

Reply:
Should be “your original message” at the end and “your post” not thred